app/(dashboard)/dashboard/interviews/_actions/deleteZipFromUploadThing.ts

```diff
@@ -0,0 +1,14 @@
+'use server';
+
+// We need to delete the exported data from UploadThing after the user has downloaded it.
+// This is to ensure that we are not storing any sensitive data on UploadThing for longer than necessary.
+
+import { utapi } from '~/app/api/uploadthing/core';
+
+export const deleteZipFromUploadThing = async (key: string) => {
+  const deleteResponse = await utapi.deleteFiles(key);
```

Comment:
There's no security on this action, as it doesn't require auth. Doesn't this mean that a malicious actor could spam the endpoint and try to delete data?

Its obviously unlikely, because file keys are UUIDs, but it still seems important...

Reply:
Good point, just added auth check here: https://github.com/complexdatacollective/Fresco/pull/113/commits/e0929fd6539203839879bb54d2e738dfcdb42a8b

Reply:
Sorry about this, here's the proper auth check: https://github.com/complexdatacollective/Fresco/pull/113/commits/db1b8d48cde89cb2ad1dd68a7e07eb037dd159da